examples/megacli.cpp

```diff
@@ -2607,6 +2623,8 @@ autocomplete::ACN autocompleteSyntax()
     p->Add(exec_treecompare, sequence(text("treecompare"), localFSPath(), remoteFSPath(client, &cwd)));
 #endif
     p->Add(exec_querytransferquota, sequence(text("querytransferquota"), param("filesize")));
+    p->Add(exec_fingerprintsumsizes, sequence(text("fingerprintsumsizes")));
```

Comment:
Perhaps we can call it `getcloudstorageused` which is more meaningful for a non-familiar user :)

Reply:
ok, will do